addons/html_editor/static/tests/caption.test.js

```diff
@@ -218,7 +218,7 @@ test("loading an image with a caption embeds it", async () => {
 });
 
 test.tags("focus required");
-test("clicking the caption button on an image with a caption removes the caption", async () => {
+test.skip("clicking the caption button on an image with a caption removes the caption", async () => {
```

Comment:
About the commit message: say "next commit" instead of "a later commit" because otherwise it reads like "we will probably fix it one day"... ^^'